lstchain/io/lstcontainers.py

```diff
@@ -79,17 +95,19 @@ class DL1ParametersContainer(Container):
     mc_core_x = Field(None, 'Simulated impact point x position', unit=u.m)
```

Comment:
`np.nan * u.m`

Reply:
Same as above, I just made the needed changes so that, when a pedestal event is processed, the container has the same contents as when a normal shower image is processed. This was not needed, and again I see no need to bring in potential trouble just before a release.